index.html

```diff
@@ -23,165 +30,161 @@
-    </div>
-    <!-- Artwork 4 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-      <!-- Artwork 5 -->
-    </div>
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-    </div>
-    <!-- Artwork 5 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-    </div>
-    <!-- Artwork 6 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-    </div>
+          <!-- Flash Cards Start  -->        
+        <div class="FlashCards">
+          <!-- Artwork 1 -->
+          <div class="Flashcard-1 ArtData worksection multiPost">
+            <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
+              Profesional Art Printing Data
+            </h3>
+            <p class="Txt textcolor workText2 Interfont">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
```

Comment:
- [ ] Kindly give the header and the paragraph for each card `margin-left` to match the Figma design

Your design
![image](https://user-images.githubusercontent.com/35535130/166123482-1aa7a031-2430-4359-b109-634d9041b2aa.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166123488-7a3940c1-d402-48b5-85ea-02cf2cebbcb6.png)